consentv2.go

```diff
@@ -449,8 +449,9 @@ func parseRange(b *bitReader, validateOnly bool) (map[int]bool, int, error) { //
 	if !ok {
 		return nil, 0, ErrUnexpectedEnd
 	}
-	vendors := make(map[int]bool)
-	for i := 0; i < int(numEntries); i++ {
+	vendors := map[int]bool{}
```

Comment:
```suggestion
	vendors := make(map[int]bool)
```
We use make when we initialize maps, and slices. Could you please revert it?